core/src/test/java/tachyon/command/TFsShellTest.java

```diff
@@ -80,9 +85,9 @@ public final void before() throws IOException {
 
   @Test
   public void catDirectoryTest() throws IOException {
-    String[] command = new String[] {"mkdir", "/testDir"};
+    String[] command = new String[] { "mkdir", "/testDir" };
```

Comment:
We have mixed formatting for this case. Currently if I run it through my formatter there are no extra spaces. I can revert these changes since it doesn't seem like we have decided on a formatting method.